torch/_inductor/lowering.py

```diff
@@ -2296,6 +2296,26 @@ def clone(x, *, memory_format=None):
     )
 
 
+def clone_preserve_reinterpret_view(x):
```

Comment:
I know there are some other clone while preserving strides type functions floating around, might be worth a checking to see if we could consolidate any of them.